frontend/src/Student/components/ShowCard.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+import PropTypes from 'prop-types'
+import Moment from 'react-moment'
+import FaPlusCircle from 'babel-loader!react-icons/fa/plus-circle'
+import { Button, ButtonGroup, Row, Col } from 'reactstrap'
+
+const Card = styled.div`
+  background-color: #f8f9fa;
+  border-radius: 5px;
+  margin-top: 15px;
+  padding: 10px;
+  width: 100%;
+`
+
+const ShowCard = (props) => (
+  <Card>
+    <Row>
+      <Col>
+        <h2>{props.name}</h2>
+      </Col>
+      <Col className="text-right">
+      <div>
+        <h5>{props.entries ? props.entries.length : 0}/{props.entryCap} submissions</h5>
+      </div>
+      <div>
+        Submission Ends: <Moment format='YYYY/MM/DD'>{props.entryEnd}</Moment>
```

Comment:
I think they should be submissions as that's what Nanette usually uses, but @robmcl4 @kayladavis opinions?

Reply:
Also, I don't think react-moment would be valid anymore if we decided to go with the glance able format

Reply:
I agree with "submissions". At some point we just chose entries as types because it was slightly shorter of a word.

Also, I agree with Bill for the date format and wording.